File: src/hello.ts

import { Timestamp } from 'google-protobuf/google/protobuf/timestamp_pb';
// import { Foo } from 'example_ts/test/foo';
import { HelloMessage } from 'example_ts/protos/hello_pb';



var msg = new HelloMessage();
msg.setGreeting("Hello");

var ts = new Timestamp();
ts.fromDate(new Date());

msg.setCreateTime(ts);

console.log("msg = ", msg.toObject());
// console.log("foo = ", new Foo());
console.log("bar");

// new Foo().setBar(123);

// msg.foo = "123";

File: src/hello.py
import google.protobuf.timestamp_pb2 as timestamp_pb2
from protos.hello_pb2 import HelloMessage

from datetime import datetime

msg = HelloMessage(greeting = "hello", create_time = timestamp_pb2.Timestamp())
msg.create_time.FromDatetime(datetime.utcnow())
# msg.create_time.GetCurrentTime()

print("Msg = %r" % msg)
